README.md

```diff
@@ -25,6 +25,13 @@ TPM emulator is a program that runs in the deamon to mimic TPM commands.
 
 ## Prerequisites
 
+### Required Package
+Rust-keylime agent rust code implementation require the following packages to both compile time and run time.
```

Comment:
Please note that these are *Fedora* packages.  (If you want to include the Debian names, that would also be fine.)  Formatting this as a `dnf install ...` would also be helpful.